app/javascript/components/episodes/Episode.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import axios from 'axios';
 import YouTube from 'react-youtube';
+import { Link } from 'react-router-dom'
+
+import { Button } from 'antd'
+
+
```

Comment:
remove redundant lines

Reply:
👯‍♂️

Reply:
![photo_2019-08-31 17 55 23](https://user-images.githubusercontent.com/25904193/64065634-87bb6f00-cc18-11e9-8556-fde46ded1fab.jpeg)

Reply:
feel free to comment and share